src/main/java/net/quedex/api/common/ListenerException.java

```diff
@@ -0,0 +1,20 @@
+package net.quedex.api.common;
+
+public class ListenerException extends Exception {
```

Comment:
Add documentation explaining when this exception is thrown and what it means. Also add relevant documentation in error listener (what exceptions should be expected).